packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -75,25 +75,29 @@ const ExportButton = styled(Button).attrs({
   variant: 'outlined',
 })`
   font-size: 0.6875rem;
+  margin: 0 1rem;
 `;
 
 const Title = styled(Typography)`
   color: white;
   font-weight: 400;
   font-size: 1.625rem;
   line-height: 1.4;
+  padding: 0 1rem;
 `;
 
 const DashboardItemsWrapper = styled.div<{
   $isExpanded: boolean;
 }>`
+  padding-left: ${({ $isExpanded }) => ($isExpanded ? '2rem' : '0')};
+  padding-right: ${({ $isExpanded }) => ($isExpanded ? '2rem' : '0')};
```

Comment:
You could probably simplify this to be `padding: ${({ $isExpanded }) => ($isExpanded ? '0 2rem' : '0')}`